### ISSUE TITLE
Question about volume control

### ISSUE THREAD

**Issue by jworozco** [snhirsch/katana-midi-bridge#1]
Hi, 

I am using FCB1010 and I i was able to set it up for preset and fx on/off, but still the manual does not show how to setup volume and expression pedals. I briefly reviewed your description and you mention CC 70, is that the one that could be used as Master Volume? 

What CC would you setup as general second expression pedal? (for Wha for example)

rgds,

wallo

**Comment by snhirsch** [comment]
Hi, wallo

At the present time, only CC#70 is mapped to sysex (master volume). I 
intend to greatly expand the available mappings, but haven't had time 
yet to do so.  Codesmart and I have very recently agreed to the 
definition of the MIDI API for CC --> sysex, so that obstacle is out of 
the way.  Thanks for your interest in the code and keep watching for 
updates.  Might be a month or so before I can get it filled out a bit more.

**Comment by jworozco** [comment]
ok thanks, will be waiting for it ;)

**Comment by jworozco** [comment]
By The way, i programmed FCB 1010 to send CC70 on the first expression pedal and CC 7 on the second to try to control volume, however i am not getting any volume change... let me know if you are able to control volume...

**Comment by snhirsch** [comment]
Volume control works just fine - I use it regularly.  Are you able to 
get basic function like program-change to work?  If so, please double 
check the MIDI channel for expression pedal 1.

I'm puzzled by your reference to CC#7 on the second pedal.  What do you 
expect that to do?  I don't have the code in front of me (at work) but I 
don't recall mapping CC#7 to anything.

**Comment by jworozco** [comment]
Yes, Present Change and Fx on/off works fine for me, but volume is not working for me. I will double check.

I took the reference for CC#7 from the General Midi standard, if you see there it is usually attached to volume control for the channel. (Look for Control Events here: https://en.wikipedia.org/wiki/General_MIDI)

**Comment by snhirsch** [comment]
I see.  I have not implemented any bindings for CC#7.

If PC and EFX toggle is operational I can only guess you have something 
misconfigured in the FCB1010.

**Comment by jworozco** [comment]
No luck, i tried using cc#70 but not getting any volume change.
I am using FCB1010 with the EurekaPROM PP mode, are you using the default original PROM from Behringer?

**Comment by snhirsch** [comment]
Hi.  If you don't mind I'd like to reopen this issue until we get you sorted out.  There are a couple of other folks using the bridge successfully (volume and all), so we need to get a handle on why your environment behaves differently.  

I am using the UnoChip 1.0.4, but I doubt that in itself has anything to do with your problem.  If you are certain that pedal #1 is sending to CC#70 on the appropriate channel, then I need to put some debugging statements in the code and have you try running with that.  Will try to find time in the next day or so.

**Comment by jworozco** [comment]
ok sure, thanks for looking at this, i will re-try the configs just to be sure... if i get it working i will let you know so that you don't have to spend extra time on it...
